src/client.cpp

```diff
@@ -337,7 +329,7 @@ void CClient::CreateServerJitterBufferMessage()
 void CClient::OnCLPingReceived ( CHostAddress InetAddr, int iMs )
 {
     // make sure we are running and the server address is correct
-    if ( IsRunning() && ( InetAddr == Channel.GetAddress() ) )
+    if ( Channel.IsEnabled() && ( InetAddr == Channel.GetAddress() ) )
```

Comment:
This isn't about the channel being enabled. It's about whether the client is running.  It's a different intent.

Reply:
I think the initial PR confused active channel and active client: https://github.com/jamulussoftware/jamulus/pull/2550#discussion_r835643759

This difference needs documentation. IMO: Add (like for JSON-RPC) respective technical documentation before the respective methods. As end goal: Write them for every function, for now: Write docs while changing.